src/contexts/walletconnect.tsx

```diff
@@ -313,10 +317,13 @@ export const WalletConnectContextProvider: FC = ({ children }) => {
 
         if (!errorMessage.includes('Pairing already exists')) {
           dispatch(walletConnectPairingFailed(errorMessage))
+          posthog.capture('Error - Could not pair with WalletConnect')
+        } else {
+          posthog.capture('Error - Could not pair with WalletConnect: Pairing already exists')
```

Comment:
An issue that I see with this is that we'll spam PostHog a bit... I am not sure if it's our implementation problem or a WC problem, but I very often get these errors in the console, without affecting the normal usage of the wallet.

Reply:
So let's start without it. As a first step, let's start with the number of successful WalletConnect sessions only, very interesting metric.